terraform/providers.tf

```diff
@@ -12,8 +12,8 @@ terraform {
   }
 
   backend "s3" {
-    encrypt = "true"
-    region  = "us-gov-west-1"
+    encrypt      = true
+    use_lockfile = true
```

Comment:
@anna-m-gsa I just learned about this today, in case you hadn't seen it yet. https://developer.hashicorp.com/terraform/language/backend/s3#use_lockfile